examples/worlds/world_exporter.sdf

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" ?>
+<!--
+
+This world will load the WorldExporter plugin, which is designed to convert
+a world to a single Collada file. When this world is run, a subdirectory
+named after the world is created. Within this subdirectory is the mesh and
+materials for the world. After a world with the SDF WorldExporter is run,
+just press CTRL-C to quit Ignition Gazebo.
+
+This example just exports simple shapes into a Collada file. To run use:
+
+    ign gazebo -v 4 world_exporter.sdf
```

Comment:
I suggest running only the server, because there's no `SceneBroadcaster`:

```suggestion
    ign gazebo -v 4 -s world_exporter.sdf
```

Reply:
Updated in 29c0a44f5e667191fc0dd80f2ca533de2d4f51c4